bin/triton-translate.cpp

```diff
@@ -94,11 +97,42 @@ LogicalResult tritonTranslateMain(int argc, char **argv,
   llvm::cl::ParseCommandLineOptions(argc, argv, toolName);
 
   mlir::MLIRContext context;
+  context.disableMultithreading();
   auto module = loadMLIRModule(inputFilename, context);
   if (!module) {
     return failure();
   }
 
+  // TODO: make these parameters
```

Comment:
Where is this diff coming from?

Reply:
This was from Goran's PR that I included accidentally. Removed.